---
layout: page
title: "Research"
date: 
modified:
excerpt:
image:
  feature:
---

### Population and host-microbiome dynamics:  What is the role of phages in human gut microbial community and obesity? ###

![Dieting Mice](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/mice.jpg)

There has been a growing appreciation of the fact that the human body is a microbially dominated ecosystem and that these microbes play a significant role in disease prevention as well as initiation. Much research has been directed at understanding how human gut microbial communities behave as a mediator that can predispose a host to diet-induced obesity. However, phages (viruses which infect bacteria) have been largely ignored in these studies even though they make up half of the gut population. My research uses metagenomic sequencing combined with computational methods to characterize the gut bacterial and viral communities in mice and the response of these communities to diet.  

### Understanding ecosystem services:  What are the microbial drivers of carbon cycling? ###

![COBS site](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/cobs.jpg)

Soils represent the most challenging ecosystem for microbial studies because of its extraordinary high diversity and structural complexity.  Similar to their role in our gut systems, microbial communities drive carbon cycling in the soil. To enable the usage of omic-based approaches and overcome the complexity of the soil, my research leverages “experimentally partitioned” soils comprised of sieved soil aggregates of varying sizes. The experimental separation of the physical soil structure into its constituent aggregates both reduces the complexity of the soil system and provides a scale that is consistent with microbiology and tractable to target microbial processes. 

### Enabling computational methods:  How can we access biodiversity in complex systems at the field-scale with available technologies? ### 

![Puzzle](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/puzzle.jpg)

Our ability to characterize complex communities is limited by two major factors:  how accurately we can sample a diverse community and how completely we can characterize this sample. While technological advances have greatly expanded or ability to sample communities, they have simultaneously ushered in new challenges pertaining to our inability to effectively analyze the results, particularly in complex environments. Application of omic-based approaches to complex environments offers the potential to obtain deep sampling of its genomic content but at the cost of extremely high volumes of data (often greater than 1 Tb for a single sample) and considerable computational resources (> 250 Gb of RAM). Analysis of such omic-datasets without available (or accurate) references requires the de novo assembly of the raw data into its original genomes (or fragments of these genomes). Analogous to building a jigsaw puzzle, de novo assembly involves identifying the connections between short genomic fragments (akin to puzzle pieces) in order to create a picture of the microbial community (without the aid of a picture on the front of the puzzle box). This objective is made more daunting because the data typically reflects under-sampled genomic diversity (missing puzzle pieces) as well as sequencing errors (pieces that don't belong to this puzzle). We have developed and successfully implemented methods suitable for assembling previously intractable large, complex datasets allowing for adequate characterization of the results of extremely large sequencing projects.  We continue to refine these approaches and apply them to build critical assembled gene reference catalogs for exploring the functional potential of multiple ecosystems. 