---
layout: page
title: "Research"
date:
modified:
excerpt:
image:
  feature:
---
## On-going research projects ##

### Improved detection and quantification of antibiotic resistant genes and organisms in the environment

![darte](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/darte.jpg)

The global spread of antimicrobial-resistant (AMR) organisms and spread of AMR-associated genes poses a serious threat to the safety of our food and public health while being responsible for increased hospitalization and mortality of both humans and production animals.  The release of AMR genes and organisms and microbes in the environment from agricultural sources is considered a serious threat but little is known about their persistence and spread in the environment.  Current risk assessment models cannot adequately characterize and quantify the proliferation of resistance.  We are developing a sequence-based approach that will provide considerably improved detection and quantification of antibiotic resistant genes including greater breadth in the numbers of genes that can be detected, the identify of microorganisms carrying these genes, and the likely association of these genes with transfer mechanisms (Diversity of Antibiotic Resistance Genes and Transfer Elements - Quantitative Monitoring, DARTE-QM).  By demonstrating this approach in both laboratory model systems and the field, we hope to identify critical control points that may be sensitive to mitigation of emergence, spread, and persistence of resistance in the environment.   Synergistic with this effort, we also are studying the occurrence and transport of antibiotics, antibiotic-resistant bacteria (ARB), and antibiotic resistance genes (ARG) in tile-drained agricultural fields that receive manure application.  We will determine the effect of manure application timing, tillage, and patterns of rainfall/drainage on the persistence and losses of antibiotics, ARB, and ARG in soil and drainage water and determine the effects of alternative manure treatment and storage on the persistence of antibiotics, ARB, and ARG in manure.

This effort is a fantastic collaboration with Iowa State University ([Michelle Soupir](http://www.public.iastate.edu/~msoupir/)), the USDA ([Heather Allen](http://www.ars.usda.gov/pandp/people/people.htm?personid=44876) and [Tom Moorman](http://www.ars.usda.gov/pandp/people/people.htm?personid=3940)), and Grinnell College ([Shannon Hinsa](https://www.grinnell.edu/users/hinsa)).


### Understanding microbial drivers of ecosystem stability:  What are the drivers of pit foaming? ###

![piggies](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/pig_npr.JPG)

We are collaborating with other investigators at Iowa State University (as well as the University of Minnesota and University of Illinois-UC) to understand the biology and chemistry that cause pit foaming in pig manure pits.  This trapped methane can present a safety threat in barns as they can explode in the presence of a spark.  We would like to understand the mechanisms behind the foaming to reduce its volume and prevent explosions.  

### Understanding ecosystem services:  What are the microbial drivers of carbon cycling? ###

![COBS site](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/cobs.jpg)

Soils represent the most challenging ecosystem for microbial studies because of its extraordinary high diversity and structural complexity.  Similar to their role in our gut systems, microbial communities drive carbon cycling in the soil. To enable the usage of omic-based approaches and overcome the complexity of the soil, our research leverages “experimentally partitioned” soils comprised of sieved soil aggregates of varying sizes. The experimental separation of the physical soil structure into its constituent aggregates both reduces the complexity of the soil system and provides a scale that is consistent with microbiology and tractable to target microbial processes.


### Understanding environmental health and productivity:  How can we use microbial indicators to assess ecosystem services and disturbances? ###

![safewater](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/Kauai_Sign_Small.jpg)

Water quality is characterized by a suite of chemical and physical measurements, as well as the usage of bioindicators, mainly the presence of microorganisms harmful to human health.  Though informative to the quality of the water, these measurements are limited in their ability to resolve and predict sources of water quality pollution.  For example, counts of colony-forming units can provide answers to how safe water is to drink or swim but do not provide identify the source of contamination. By unifying data collections of existing state-wide monitoring efforts (e.g., Iowa DNR Lake Monitoring & ISU Limnology Laboratory) and microbial community analyses, we hope to identify microbial drivers of various water functions and ecosystem health (e.g., nutrient cycling, pathogens, non-point sources of pollution).

## Other Research Projects ##

### Enabling computational methods:  How can we access biodiversity in complex systems at the field-scale with available technologies? ###

![Puzzle](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/puzzle.jpg)

Our ability to characterize complex communities is limited by two major factors:  how accurately we can sample a diverse community and how completely we can characterize this sample. While technological advances have greatly expanded or ability to sample communities, they have simultaneously ushered in new challenges pertaining to our inability to effectively analyze the results, particularly in complex environments. Application of omic-based approaches to complex environments offers the potential to obtain deep sampling of its genomic content but at the cost of extremely high volumes of data (often greater than 1 Tb for a single sample) and considerable computational resources (> 250 Gb of RAM). Analysis of such omic-datasets without available (or accurate) references requires the de novo assembly of the raw data into its original genomes (or fragments of these genomes). Analogous to building a jigsaw puzzle, de novo assembly involves identifying the connections between short genomic fragments (akin to puzzle pieces) in order to create a picture of the microbial community (without the aid of a picture on the front of the puzzle box). This objective is made more daunting because the data typically reflects under-sampled genomic diversity (missing puzzle pieces) as well as sequencing errors (pieces that don't belong to this puzzle). We have developed and successfully implemented methods suitable for assembling previously intractable large, complex datasets allowing for adequate characterization of the results of extremely large sequencing projects.  We continue to refine these approaches and apply them to build critical assembled gene reference catalogs for exploring the functional potential of multiple ecosystems.

### Population and host-microbiome dynamics:  What is the role of phages in human gut microbial community and obesity? ###

![Dieting Mice](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/mice.jpg)

There has been a growing appreciation of the fact that the human body is a microbially dominated ecosystem and that these microbes play a significant role in disease prevention as well as initiation. Much research has been directed at understanding how human gut microbial communities behave as a mediator that can predispose a host to diet-induced obesity. However, phages (viruses which infect bacteria) have been largely ignored in these studies even though they make up half of the gut population. Our research uses metagenomic sequencing combined with computational methods to characterize the gut bacterial and viral communities in mice and the response of these communities to diet.  

## More Information ##
* [Publications](https://scholar.google.com/citations?user=ixR8YE8AAAAJ&hl=en)
* [Digital Repository](http://works.bepress.com/adina/) - though we try VERY hard to publish with open access, this link will provide access to articles that may be difficult to obtain without a paywall.
* [Howe Research Statement (Spring 2014)](./research.pdf)
* [Howe Teaching Statement (Spring 2014)](./teaching.pdf)
* [Howe CV (September 2015)](./cv.pdf) (contains links to all publications)
