---
layout: page
title: "Research"
date:
modified:
excerpt:
image:
  feature:
---
## On-going research projects ##

### Improved detection and quantification of antibiotic resistant genes and organisms in the environment

![darte](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/darte.jpg)

The global spread of antimicrobial-resistant (AMR) organisms and spread of AMR-associated genes poses a serious threat to the safety of our food and public health while being responsible for increased hospitalization and mortality of both humans and production animals.  The release of AMR genes and organisms and microbes in the environment from agricultural sources is considered a serious threat but little is known about their persistence and spread in the environment.  Current risk assessment models cannot adequately characterize and quantify the proliferation of resistance.  We are developing a sequence-based approach that will provide considerably improved detection and quantification of antibiotic resistant genes including greater breadth in the numbers of genes that can be detected, the identify of microorganisms carrying these genes, and the likely association of these genes with transfer mechanisms (Diversity of Antibiotic Resistance Genes and Transfer Elements - Quantitative Monitoring, DARTE-QM).  By demonstrating this approach in both laboratory model systems and the field, we hope to identify critical control points that may be sensitive to mitigation of emergence, spread, and persistence of resistance in the environment.   Synergistic with this effort, we also are studying the occurrence and transport of antibiotics, antibiotic-resistant bacteria (ARB), and antibiotic resistance genes (ARG) in tile-drained agricultural fields that receive manure application.  We will determine the effect of manure application timing, tillage, and patterns of rainfall/drainage on the persistence and losses of antibiotics, ARB, and ARG in soil and drainage water and determine the effects of alternative manure treatment and storage on the persistence of antibiotics, ARB, and ARG in manure.

This effort is a fantastic collaboration with Iowa State University ([Michelle Soupir](http://www.public.iastate.edu/~msoupir/)), the USDA ([Heather Allen](http://www.ars.usda.gov/pandp/people/people.htm?personid=44876) and [Tom Moorman](http://www.ars.usda.gov/pandp/people/people.htm?personid=3940)), and Grinnell College ([Shannon Hinsa](https://www.grinnell.edu/users/hinsa)).


### Understanding environmental health and productivity:  How can we use microbial indicators to assess ecosystem services and disturbances? ###

![safewater](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/Kauai_Sign_Small.jpg)

We use a systems approach to identify genetic and environmental factors controlling the occurrence of harmful algal blooms (HABs) in Iowa's recreational lakes. We hypothesize that HAB ecology and cyanotoxin production are the predictable result of environmental factors, that the underlying genetic markers for cyanotoxin production are taxonomically controlled, and that incipient cyanotoxin-degrading microbes are present during HABs. The proposed research aims to test these hypotheses through development of monitoring and predictive tools to target future cyanotoxin monitoring and mitigation to the highest-risk recreational waters.  The following objectives will be accomplished: conduct an integrated meta-analysis of physicochemical parameters and microbiome analyses of Iowa's HAB-impacted recreational waters to develop a predictive model of HAB occurrence; develop scalable tools that can be used to rapidly monitor HABs and identify when additional cyanotoxin monitoring is necessary; identify emerging cyanotoxins within Iowa's lakes and determine the freshwater HAB species linked to these toxins and the genetic systems that control toxin production; and identify and evaluate novel toxin-degraders for the mitigation of HAB cyanotoxins.


### Understanding ecosystem services:  What are the microbial drivers of soil health in bioenergy crops? ###

![COBS site](https://raw.githubusercontent.com/germs-lab/germs-lab.github.com/master/images/cobs.jpg)

 Using a data-driven and integrated modeling framework, we collaborate with researchers who will develop the predictive capability to determine which feedstock combinations, regions and land types, market conditions, and bioproducts have the potential to support the ecologically and economically sustainable displacement of fossil fuels. Soils represent the most challenging ecosystem for microbial studies because of its extraordinary high diversity and structural complexity.  Similar to their role in our gut systems, microbial communities drive nutrient cycling in the soil. In particular, our area of emphasis will be to obtain a mechanistic understanding of the plant, soil, microbe, and climate interactions that underlie the productivity and ecosystem services of different feedstocks.

## More Information ##
* [Publications](https://scholar.google.com/citations?user=ixR8YE8AAAAJ&hl=en)
* [Digital Repository](http://works.bepress.com/adina/) - though we try VERY hard to publish with open access, this link will provide access to articles that may be difficult to obtain without a paywall.
* [Howe Research Statement (Spring 2014)](./research.pdf)
* [Howe Teaching Statement (Spring 2014)](./teaching.pdf)
* [Howe CV (September 2015)](./cv.pdf) (contains links to all publications)
